languages/javascript/exercises/concept/fruit-picker/fruit-picker.spec.js

```diff
@@ -0,0 +1,120 @@
+import {
```

Comment:
```suggestion
// @ts-check

import {
```

See if vscode still likes this file if you add this